ws-nextjs-app/pages/[service]/live/[id]/Stream/styles.tsx

```diff
@@ -21,6 +21,18 @@ export default {
         paddingTop: `${spacings.DOUBLE}rem`,
       },
     }),
+  headingNoContributors: ({ mq, spacings, fontSizes }: Theme) =>
+    css({
+      padding: `${spacings.DOUBLE}rem 0 ${spacings.DOUBLE}rem`,
+      [mq.GROUP_3_MIN_WIDTH]: {
+        ...fontSizes.doublePica,
+        paddingTop: `${spacings.TRIPLE}rem`,
+        paddingBottom: `${spacings.TRIPLE}rem`,
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        paddingTop: `${spacings.DOUBLE}rem`,
+      },
+    }),
```

Comment:
Theres an Emotion CSS trick you could do here to prevent duplicating a lot of the same styles. 

You could declare the one style that is different between `heading` and `headingNoContributors`, which is:

```
    css({
      [mq.GROUP_3_MIN_WIDTH]: {
        paddingBottom: `${spacings.TRIPLE}rem`,
      },
    }),
```  
Then on the actual element, you can set the base styling to always be `styles.heading` and then conditionally add in this new style when there is no contributors. It would look something like:
```
      <Heading
        css={[styles.heading, !contributors && styles.headingNoContributors]}
        level={2}
      >
        Live Reporting
      </Heading>
```
Its an array, so the first item is the base set of styles, then the second item is either `null` or `styles.headingNoContributors` when there are no contributors. Emotion will merge and combine these styles together.